src/VPolygon.jl

```diff
@@ -0,0 +1,112 @@
+import Base.<=
+
+# Polygons in vertex representation
+export VPolygon, tovrep, vertices_list, is_contained
+
+"""
+    VPolygon <: LazySet
+
+Type that represents a polygon by its vertices.
+
+### Fields
+
+- `vertices_list` -- the list of vertices
+
+### Notes
+
+The constructor of `VPolygon` runs a convex hull algorithm, and the given vertices
+are sorted in counter-clockwise fashion. If you don't want to take the
+convex hull, set the `apply_convex_hull=false` flag when instantiating the constructor.
+"""
+struct VPolygon <: LazySet
+    vertices_list::Vector{Vector{Float64}}
+
+    function VPolygon(vertices_list; apply_convex_hull=true, algorithm="andrew")
+        if apply_convex_hull
+            return new(convex_hull(vertices_list, algorithm=algorithm))
+        else
+            return new(vertices_list)
+        end
+    end
+    #VPolygon(vertices_list; apply_convex_hull=true; algorithm=) = new(convex_hull(vertices_list))
+end
+VPolygon() = VPolygon([])
+
+
+"""
+    vertices_list(P)
+
+Return the list of vertices of a convex polygon in vertex representation.
+
+### Input
+
+- `P` -- a polygon given in vertex representation
+
+### Output
+
+List of vertices as an array of vertex pairs, Vector{Vector{Float64}}.
+"""
+function vertices_list(P::VPolygon)::Vector{Vector{Float64}}
+    return P.vertices_list
+end
+#=
```

Comment:
Why is this kept here? It is already added in `HPolygon.jl`.
Also: Should we write `plot_polygon` everywhere (with lowercase "p")?

Reply:
why not? it's the `vertices_list` of a `VPolygon`.

the commented code is garbage; i'll fix it.

yep, `plot_polygon` everywhere.

PS: still i haven't worked out the new support function for this `VPolygon` type.

Reply:
Yes, I meant the code that follows, Github shows the three preceding lines...